src/main/api/2/studio-api-2.yaml

```diff
@@ -858,7 +858,7 @@ paths:
         '500':
           $ref: '#/components/responses/InternalServerError'         
 
-  /users/{id}/set_password:
+  /users/me/set_password:
```

Comment:
Why `/me` and not `/` since the User ID is not known at that time? While it's possible to assume it's the current user, which will be the case most of the time, they're not yet authenticated.